src/pages/HomePage/pages/LifeCenter/LifeCenter.tsx

```diff
@@ -0,0 +1,146 @@
+import { useEffect, useState } from "react";
+// import PageOutline from "../../Components/PageOutline";
+import TabSelection from "@/pages/HomePage/Components/reusable/TabSelection";
+import { Button } from "@/components";
+import Modal from "@/components/Modal";
+import { HeaderControls } from "@/components/HeaderControls";
+import LifeCenterForm from "./components/LifeCenterForm";
+import LifeCenterCard from "./components/LifeCenterCard";
+import useWindowSize from "@/CustomHooks/useWindowSize";
+import { useFetch } from "@/CustomHooks/useFetch";
+import { api } from "@/utils/api/apiCalls";
+import GridComponent from "../../Components/reusable/GridComponent";
+
+export type LifeCenterType = {
+  name: string;
+  description: string;
+  location: string;
+  meeting_dates: string[];
+  num_of_members: number;
+  num_of_souls_won: number;
+  id?: string;
+};
+
+export default function LifeCenter() {
+  const [selectedTab, setSelectedTab] = useState("Life Center");
+  const [showModal, setShowModal] = useState(false);
+  const windowSize = useWindowSize();
+  const { data } = useFetch(api.fetch.fetchAllLifeCenters);
+
+  const handleTabSelect = (tab: string) => {
+    setSelectedTab(tab);
+  };
+
+  const [lifeCenters, setLifcenters] = useState<LifeCenterType[]>([]);
+  useEffect(() => {
+    if (data?.data.length) {
+      setLifcenters([...data.data]);
+    }
+  }, [data]);
+
+  const toggleModal = () => {
+    setShowModal((prev) => !prev);
+  };
+
+  const [initialValues, setInitialValues] = useState<LifeCenterType>({
```

Comment:
please move this to the bottom of the code as constant 
Declaring the initial state object outside the component makes the code cleaner, avoids recreating the object on every render, and allows you to reuse the same default values in multiple places (such as resetting the form).

you can check members or visitors or assets on how I did that